roles/installer/defaults/main.yml

```
@@ -278,10 +278,12 @@ init_container_extra_commands: ''
 init_container_extra_volume_mounts: ''
 
 ee_images:
-  - name: AWX EE (latest)
-    image: quay.io/ansible/awx-ee:latest
+  - name: "AWX EE (default)"
+    image: "quay.io/ansible/awx-ee:{{ _image_version }}"
```

@TheRealHaoLiu 
Thanks for updating, perhaps this line is incorrect.

```suggestion
    image: quay.io/ansible/awx-ee:latest
```

---

i still want to have versioned image there instead of floating tag "latest"

changing `AWX EE (latest)` to `AWX EE (default)` to denote that its no longer `latest`

do you think we should still have an `AWX EE (latest)`?

---

ok ur right, lets put back the latest for backward compatibility 

the main thing I'm trying to do at least is for the controlplane-ee to STOP using latest